### ISSUE TITLE
what to do if we want to calculate the scores for video sessions which are greater than 5 min duration?

### ISSUE THREAD

**Issue by Ayushi-0803** [itu-p1203/itu-p1203#36]

**Comment by Ayushi-0803** [comment]
what can we do if the session which we are playing is longer than 5 min duration? how can we possibly break it into segments of 5 sec and take the average of those ?

**Comment by slhck** [comment]
Thanks for your question. There is no formal procedure for evaluating the MOS (using an algorithm) for sessions of such length using this model. The P.1203.3 model has been validated only for inputs of up to 5 min length — that is the duration of the test sequences that have been shown to the test subjects during subjective evaluation. Any extrapolation beyond that would be inaccurate, although of course nothing prevents you from running the model on, say, sequences of 10 minutes length.

The P.1203.1 and P.1203.2 MOS values (per-second video and audio quality scores, O22 and O21, respectively) are of course still valid for input sequences of arbitrary length, because they consider at most 20 seconds worth of input to determine the instantaneous quality.

But keep in mind that the overall MOS (O46) is a _retrospective_ rating of the subjective quality of the past 5 minutes, including any initial loading delay, quality variation, or stalling happening before. When a sequence is longer than that, e.g., an hour-long movie, humans tend to forget what happened in the beginning (even if there were issues) due to the peak-end and recency effects (Kahneman et al.). We don't know whether a model prediction holds up when applied to a longer video.

What you *could* do is calculate the O46 MOS for artificial chunks of the input video, e.g., you generate overlapping sequences of 5 minutes, in a sliding window type of manner, then average these per-5-minute scores.

**Comment by Ayushi-0803** [comment]
Thanks for the answer...
When we say durations of upto 5 minute long ....do we also include the stalling impact duration in it? say the video ran for 4 min but it stalls for 30 sec on the whole so thw whole duration will be taken to be 4 min 30 sec right?

**Comment by slhck** [comment]
Yeah, if I am not mistaken, it means "5 minutes of media duration" excluding stalling.

**Comment by Ayushi-0803** [comment]
ITU-T P.1203.3: Audiovisual integration and integration of final score, reflecting remembered
quality for viewing sessions between 30 s and 5 min duration 
it says viewing sessions of upto 5 min duration doesnt viewing session include stalls also?

**Comment by slhck** [comment]
Yes, in principle you are right. However, I do not remember the exact configuration of the training databases for 5 minutes, but the clips may have been a bit shorter or longer than 5 minutes in total. Their nominal/average duration was 5 minutes; the longest one was 308 seconds, and stalling was never longer than 15 seconds I think.

**Comment by Ayushi-0803** [comment]
Thanks !
